wire-ios-sync-engine/Source/SessionManager/SessionManager+Backup.swift

```diff
@@ -111,8 +113,16 @@ extension SessionManager {
         guard BackupFileExtensions.allCases.contains(where: { $0.rawValue == location.pathExtension }) else { return completion(.failure(BackupError.invalidFileExtension)) }
 
         SessionManager.workerQueue.async(group: dispatchGroup) { [weak self] in
-            guard let `self` = self else { return }
+            guard let `self` = self else {
+                completion(.failure(NSError(code: .unknownError, userInfo: ["reason": "memory allocation"])))
```

Comment:
"memory allocation" sounds confusing, what about "SessionManager.self is `nil`" or "SessionManager instance destroyed"?